connect/mirror/src/test/java/org/apache/kafka/connect/mirror/MirrorMakerConfigTest.java

```diff
@@ -52,10 +52,10 @@ public void testClusterConfigProperties() {
             "replication.factor", "4"));
         Map<String, String> connectorProps = mirrorConfig.connectorBaseConfig(new SourceAndTarget("a", "b"),
             MirrorSourceConnector.class);
-        assertEquals("source.cluster.bootstrap.servers is set",
-            "servers-one", connectorProps.get("source.cluster.bootstrap.servers"));
-        assertEquals("target.cluster.bootstrap.servers is set",
-            "servers-two", connectorProps.get("target.cluster.bootstrap.servers"));
+        assertEquals("source.bootstrap.servers is set",
```

Comment:
I was not expecting to see changes in existing tests if we are not breaking compatibility, `source.cluster.bootstrap.servers` should continue working and still be tested

Reply:
That will require more changes. This does not break for the mirrorMaker driver main config. I guess it does for on Connect though

Reply:
I think I have achieved what we want. Explicitly setting "source." prefix for props starting with consumer|producer|admin and setting "source.cluster." otherwise. All tests passed. The code runs fine on my infra.